src/utils/interfaces.ts

```diff
@@ -1,11 +1,12 @@
 // This is a file of TS interfaces that get used in multiple places
 // So they live here so you can import them!
 
-interface Card {
+interface CardObj {
   id: number;
   isVisibleToAll: boolean;
   wordString: string;
   wordId: number;
+  word: WordObj;
```

Comment:
word needs to be included bc it contains the WordObj which is different from the wordString. We probably don't need to have the word obj on the cardObj at all, but this is how we previously structured it. maybe can refactor once we have MVP complete.